### ISSUE TITLE
Nuitka looses sys.prefix and sys.exec_prefix after build

### ISSUE THREAD

**Issue by zloidemon** [Nuitka/Nuitka#240]
Linux version:

```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.4 LTS
Release:        16.04
Codename:       xenial
```
Python version:

```
$ python -V
Python 3.5.2
```

Both version lates from pypi and from developer branch has errors:

```
0.6.2rc3
Python: 3.5.2 (default, Nov 23 2017, 16:37:01)
Executable: /home/vg/tmp/.venv/bin/python
OS: Linux
Arch: x86_64
```

Test code
```
$ cat test.py
import setproctitle

setproctitle('test')
```

Run builded app:
```
$ ./test.bin
Traceback (most recent call last):
  File "/home/vg/tmp/test.py", line 1, in <module>
    import setproctitle
ImportError: No module named 'setproctitle'
```

MacOS works fine same code.

**Comment by kayhayen** [comment]
You failed to include the command line used to produce the binary. 

You likely failed to use Nuitka with the python binary where the package is installed, or you believe that accelerated binaries run outside of the environment where they are created.

**Comment by kayhayen** [comment]
As an aside, your example is calling the module, probably not what you meant.

**Comment by zloidemon** [comment]
I'm sorry missed add command to build

```
python -m nuitka --follow-imports test.py
```

**Comment by kayhayen** [comment]
And `python test.py` actually works? If so, I would be interested in the output for Nuitka given for `--explain-imports` command line flag, which should demonstrate how it doesn't find things. Since it's an extension module, maybe there is something special there.

**Comment by zloidemon** [comment]
Import is enough to reproduce issue

```
(.venv) vg:~/tmp$ cat test.py
import setproctitle
```

Output:
```
(.venv) vg:~/tmp$ python test.py
(.venv) vg:~/tmp$ python -m nuitka --follow-imports test.py  --explain-imports
findModule: Enter to search 'site' in package None level 0.
_findModule: Enter to search 'site'.
_findModuleInPath: Enter site in None
findModule: Found absolute imported module 'site' in filename '/home/vg/tmp/.venv/lib/python3.5/site.py':
findModule: Enter to search 'setproctitle' in package None level 0.
_findModule: Enter to search 'setproctitle'.
_findModuleInPath: Enter setproctitle in None
_findModuleInPath: Using search path ['/home/vg/tmp', '/home/vg/tmp', '.', '/home/vg/tmp/.venv/lib/python35.zip', '/home/vg/tmp/.venv/lib/python3.5', '/home/vg/tmp/.venv/lib/python3.5/plat-x86_64-linux-gnu', '/home/vg/tmp/.venv/lib/python3.5/lib-dynload', '/usr/lib/python3.5', '/usr/lib/python3.5/plat-x86_64-linux-gnu', '/home/vg/tmp/.venv/lib/python3.5/site-packages'] for None
Candidates OrderedSet([('/home/vg/tmp/.venv/lib/python3.5/site-packages', 1, '/home/vg/tmp/.venv/lib/python3.5/site-packages/setproctitle.cpython-35m-x86_64-linux-gnu.so')])
_findModuleInPath: _findModuleInPath2 gave /home/vg/tmp/.venv/lib/python3.5/site-packages/setproctitle.cpython-35m-x86_64-linux-gnu.so
findModule: Found absolute imported module 'setproctitle' in filename '/home/vg/tmp/.venv/lib/python3.5/site-packages/setproctitle.cpython-35m-x86_64-linux-gnu.so':

(.venv) v:~/tmp$
(.venv) vg:~/tmp$ ./test.bin
Traceback (most recent call last):
  File "/home/vg/tmp/test.py", line 1, in <module>
    import setproctitle
ImportError: No module named 'setproctitle'
```

**Comment by zloidemon** [comment]
site.py's sent to email from github profile. I usually create venv like this lines:

```
$ export LC_ALL=C
$ virtualenv -m python3 .venv
```

**Comment by kayhayen** [comment]
So, `sys.prefix` and `sys.exec_prefix` are used in this `site.py` to expand the site path, and that is missing, so basically that form of virtualenv is not supported for accelerated mode.

**Comment by kayhayen** [comment]
For the venv, there is not even a `site.py` which makes it work. However, much like `sys.executable` that is already preserved, I can add those easily too.

**Comment by zloidemon** [comment]
> @zloidemon I managed to reproduce it that way. I was before using `python3.5 -m venv` which is included in Python for some time now. Obviously the `site.py` needs some special trick that fails to work compiled, I will look into it. However, using `venv` is a workaround you can try.

Ok, i'll try `python3.5 -m venv`

**Comment by kayhayen** [comment]
I have pushed a fix to factory branch, you can try it if you want to:

http://nuitka.net/doc/factory.html

**Comment by kayhayen** [comment]
Preserving `sys.prefix` seemed good enough for `site.py` to finds its way. Please confirm that it's working if you can with your `virtualenv`.

**Comment by zloidemon** [comment]
> I have pushed a fix to factory branch, you can try it if you want to:
> 
> http://nuitka.net/doc/factory.html

It works. Thank you.

**Comment by kayhayen** [comment]
Released as 0.6.2
